File: C/Colleen.c
#include <stdio.h>
/*
 * Comment outside of main function
 */

void another_func()
{
	char *str = "#include <stdio.h>%c/*%c * Comment outside of main function%c */%c%cvoid another_func()%c{%c	char *str = %c%s%c;%c	printf(str, 10, 10, 10, 10, 10, 10, 10, 34, str, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10);%c}%c%cint main() {%c	/*%c	 * Comment in the main function%c	 */%c	another_func();%c	return 0;%c}%c";
	printf(str, 10, 10, 10, 10, 10, 10, 10, 34, str, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10);
}

int main() {
	/*
	 * Comment in the main function
	 */
	another_func();
	return 0;
}

File: python/Colleen.py
# Comment outside

def another_func():
    str = "# Comment outside{0}{0}def another_func():{0}    str = {2}{1}{2}{0}    print(str.format(chr(10), str, chr(34))){0}{0}if __name__ == '__main__':{0}    # Comment inside{0}    another_func()"
    print(str.format(chr(10), str, chr(34)))

if __name__ == '__main__':
    # Comment inside
    another_func()
